.github/workflows/ci.yml

```
@@ -35,6 +35,8 @@ jobs:
           prefix: ${{ github.event.repository.name }} # cloud-file-manager
           awsAccessKeyId: ${{ secrets.AWS_ACCESS_KEY_ID }}
           awsSecretAccessKey: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          githubToken: ${{ secrets.GITHUB_TOKEN }}
+          deployRunUrl: https://models-resources.concord.org/cloud-file-manager/__deployPath__/index.html
```

This would be a better URL I think:
```suggestion
          deployRunUrl: https://models-resources.concord.org/cloud-file-manager/__deployPath__/examples/
```